### PR TITLE
Issue/quick 103 rebase fixed

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -52,7 +52,7 @@ task<Test>("cleanUpAfterBamboo").apply {
 
 dependencies {
     testCompile(project(":virtual-users"))
-    compile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
+    testCompile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
```
Comment (atl-mk) [inline build.gradle.kts:55]: What is the difference between compile and testCompile?

**Review by atl-mk — APPROVED** [review]
LGTM

File: build.gradle.kts (hunk 1 above)
Comment (dagguh) [reply build.gradle.kts:55]: `compile` is available for `src/main` and `src/test`, while `testCompile` is for `src/test` only
We don't even have `src/main` in this repo, so it doesn't really matter. However, it's a good practice to limit scopes as much as possible.

**Comment by dagguh** [comment]
@kema2000 please accept the GH invite, so you'll be able to review and track these changes

**Pull request merged**